app/destination/page.js

```diff
@@ -74,41 +123,30 @@ export const Destinations = () => {
         </section>
         <section className="card">
           <h2>Possible destinations</h2>
+
           {/* TASK - React 1 week 2 */}
           {/* Add all 4 planets! Europa, Moon, Mars, Titan  */}
           {/* Use the README.md file for descriptions */}
           {/* Create a <PlanetCard /> component, which accepts the following properties: */}
           {/* name, description, thumbnail, isSelected, onAddOrRemovePlanet */}
-          <div className={styles.planetCard}>
-            <img className={styles.planetThumbnail} src="/destination/image-europa.png" alt="" />
-            <div className={styles.planetDescription}>
-              <h2>EUROPA {isPlanetSelected ? "- SELECTED" : ""}</h2>
-              <p>Lorem ipsum...</p>
-            </div>
-            <button 
-              className="roundButton" 
-              onClick={() => onAddOrRemovePlanet('Pluto', 0)}
-            > 
-              {isPlanetSelected ? "REMOVE" : "ADD PLANET"}
-            </button>
-          </div>
-          <div className={styles.planetCard}>
-            <img className={styles.planetThumbnail} src="/destination/image-europa.png" alt="" />
-            <div className={styles.planetDescription}>
-              <h2>EUROPA {isPlanetSelected ? "- SELECTED" : ""}</h2>
-              <p>Lorem ipsum...</p>
-            </div>
-            <button 
-              className="roundButton" 
-              onClick={() => onAddOrRemovePlanet('Pluto', 0)}
-            > 
-              {isPlanetSelected ? "REMOVE" : "ADD PLANET"}
-            </button>
-          </div>
+          
+          {planets.map((planet, index) => (
+                <PlanetCard
+                  key={index}
+                  name={planet.name}
+                  description={planet.description}
+                  thumbnail={planet.thumbnail}
+                  isSelected={selectedPlanets.includes(planet.name)}
+                  onAddOrRemovePlanet={onAddOrRemovePlanet}
```

Comment:
```suggestion
                  onAddOrRemovePlanet={() => onAddOrRemovePlanet(planet.name, index)}
```
This is where you should use the index of the map loop.